cloudfront/wellcomelibrary.org/edge-lambda/src/readRedirects.ts

```diff
@@ -20,10 +27,17 @@ export function readStaticRedirects(
       .parseFile<RedirectRow, RedirectRow>(fileLocation, options)
       .on('error', reject)
       .on('data', (row: RedirectRow) => {
-        if (row.libraryUrl) {
-          const lookupKey = row.libraryUrl
+        if (row.sourceUrl) {
+          if (!row.sourceUrl.startsWith(hostPrefix)) {
+            throw Error(
```

Comment:
I'm battling to see where we catch this error?

Reply:
We don't. Failure to parse the CSV will allow the error to bubble up and stop execution (this won't happen in a lambda though).